src/main/java/de/internetsicherheit/brl/bloxberg/cache/BlockDataExtractor.java

```diff
@@ -50,11 +51,11 @@ public void generateJsonFile() throws IOException {
 
     }
 
-
     public void writeOutTransactions(int blockNumber, SequenceWriter seqWriter) throws IOException {
-        BigInteger blockBigInteger = BigInteger.valueOf(blockNumber);
-        List<BlockTransaction> transactions = client.getBlockWithData(blockBigInteger).getTransactions();
-        BigInteger timestamp = client.getBlockTimestamp(blockBigInteger);
+        BigInteger blockNumberToBigInteger = BigInteger.valueOf(blockNumber);
+        Block block = client.getBlock(blockNumberToBigInteger);
```

Comment:
Why is it better not having the ```block``` variable, if for getting the timestamp we have to call: ```  BigInteger timestamp = client.getBlock(blockNumberToBigInteger).getTimestamp();``` (and also calling ```client.getBlock(blockNumberToBigInteger)``` for fetching the ```transactions```) instead of already having a variable that did the call ```client.getBlock(blockNumberToBigInteger)``` only once before?